docs/dev/connectors/index.zh.md

```diff
@@ -43,6 +43,7 @@ under the License.
  * [Amazon Kinesis Streams](kinesis.html) (source/sink)
  * [Elasticsearch](elasticsearch.html) (sink)
  * [Hadoop FileSystem](streamfile_sink.html) (sink)
```

Comment:
```suggestion
 * [FileSystem (Hadoop included) - SourceFunction API](streamfile_sink.html) (sink)
```
or something like that

Reply:
Hi @azagrebin , thanks for looking into it. I think this is not for this PR. Also the `StreamingFileSink` is based on the new `Sink API` and not `Source API`. Did I missunderstand your comment?

Reply:
Sorry, of course, I meant `SinkFunction` :) and `StreamingFileSink` is based on _old_ API right?

Reply:
Yes, I added a note in the docs of the new `FileSink` that explains the difference with the existing `StreamingFileSink`. I hope this covers your comment @azagrebin .

Reply:
My original concern was that `Hadoop FileSystem` points to `StreamingFileSink` which is not only about `Hadoop` but this PR is about `FileSink`. It is just a coincidence that I was trying to find file connector docs and found only `Hadoop` link and then noticed this PR, so my comment was more about raising awareness about this inconsistency for `StreamingFileSink` which is not a problem for the new `FileSink`.

Reply:
I see, I can update it as a separate commit then but in the same PR.